ui/src/pages/repo.tsx

```diff
@@ -180,6 +193,19 @@ function RepoWrapper({ children, id }) {
               Share
             </Button>
           }
+          forkButton={
+            <Button
+              endIcon={<ContentCopyIcon />}
+              onClick={async () => {
+                const result = await copyRepo();
```

Comment:
This is interesting; I didn't know we could await for the mutation function. Looks like we indeed can.